<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnakeMainWindow</class>
 <widget class="QMainWindow" name="SnakeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="snake.qrc">
    <normaloff>:/pixmap/SnakeHead.png</normaloff>:/pixmap/SnakeHead.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="snake_img_">
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>230</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>580</width>
        <height>230</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimum">
         <number>400</number>
        </property>
        <property name="maximum">
         <number>900</number>
        </property>
        <property name="singleStep">
         <number>32</number>
        </property>
        <property name="value">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="MapWidth">
        <property name="text">
         <string>Map width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="MapHeight">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Map height:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="width_lbl_">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>400</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="height_lbl_">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>400</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="minimum">
         <number>400</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="singleStep">
         <number>32</number>
        </property>
        <property name="value">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Difficulty:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="easy_btn_">
        <property name="text">
         <string>Easy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="intermediate_btn_">
        <property name="text">
         <string>Intermediate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hard_btn_">
        <property name="text">
         <string>Hard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="start_game_btn_">
        <property name="text">
         <string>Start game</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>231</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exit_btn_">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>start_game_btn_</tabstop>
  <tabstop>exit_btn_</tabstop>
  <tabstop>snake_img_</tabstop>
 </tabstops>
 <resources>
  <include location="snake.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>width_lbl_</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height_lbl_</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_btn_</sender>
   <signal>clicked()</signal>
   <receiver>SnakeMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_game_btn_</sender>
   <signal>clicked()</signal>
   <receiver>SnakeMainWindow</receiver>
   <slot>StartGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>StartGame()</slot>
 </slots>
</ui>
